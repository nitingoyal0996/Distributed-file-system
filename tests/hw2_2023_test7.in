create f1
create f2
create f3
create f4
create f5
create f6
create f7
create f8
create f9
create f10
create f11
create f12
create f13
create f14
create f15
create f16
create f17
create f18
create f19
create f20
create f21
create f22
create f23
create f24
create f25
ls
rm f1
ls
rm f2
rm f3
rm f4
rm f5
rm f6
rm f7
rm f8
rm f9
rm f10
ls
rm f11
rm f12
rm f13
rm f14
rm f15
rm f16
rm f17
rm f18
rm f19
rm f20
rm f21
rm f22
rm f23
rm f24
ls
rm f25
ls
exit
