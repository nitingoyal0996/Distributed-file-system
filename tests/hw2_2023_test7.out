#### File system information:
Number of blocks          : 256
Block size (Bytes)        : 256
Number of inodes          : 32
inode size (Bytes)        : 32
inodes per block          : 8
Free bitmap offset        : 2
Free bitmap size (blocks) : 1
Inode table offset        : 3
Inode table size (blocks) : 4
Max blocks per file       : 6
Data blocks offset        : 7
Data block size (blocks)  : 249
Raw block layer layout: (B: boot, S: superblock, F: free bitmap, I: inode, D: data
0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345
BSFIIIIDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[26]:./
[1]:f1
[1]:f2
[1]:f3
[1]:f4
[1]:f5
[1]:f6
[1]:f7
[1]:f8
[1]:f9
[1]:f10
[1]:f11
[1]:f12
[1]:f13
[1]:f14
[1]:f15
[1]:f16
[1]:f17
[1]:f18
[1]:f19
[1]:f20
[1]:f21
[1]:f22
[1]:f23
[1]:f24
[1]:f25
[cwd=0]%
[cwd=0]%
[25]:./
[1]:f2
[1]:f3
[1]:f4
[1]:f5
[1]:f6
[1]:f7
[1]:f8
[1]:f9
[1]:f10
[1]:f11
[1]:f12
[1]:f13
[1]:f14
[1]:f15
[1]:f16
[1]:f17
[1]:f18
[1]:f19
[1]:f20
[1]:f21
[1]:f22
[1]:f23
[1]:f24
[1]:f25
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[16]:./
[1]:f11
[1]:f12
[1]:f13
[1]:f14
[1]:f15
[1]:f16
[1]:f17
[1]:f18
[1]:f19
[1]:f20
[1]:f21
[1]:f22
[1]:f23
[1]:f24
[1]:f25
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[2]:./
[1]:f25
[cwd=0]%
[cwd=0]%
[1]:./
[cwd=0]%
